e3nn/networks.py

```diff
@@ -16,9 +16,49 @@
 
 
 class GatedConvNetwork(torch.nn.Module):
+    """A basic network architecture that alternates between Convolutions and GatedBlock nonlinearities.
+
+    This network architecture should be used if you only want to be equivariant to rotations and translations but not inversion. GatedBlock acts on irreps of SO(3), meaning it does not include parity. If you want mirror symmetry and inversion, use GatedConvParityNetwork which handles the full irreps of O(3).
+
+    Args:
+        Rs_in ([rs.TY_RS_STRICT]): Representation list of input data. Parity must be omitted or set to 0.
+        Rs_hidden ([rs.TY_RS_STRICT]): Representation list of intermediate data. Parity must be omitted or set to 0.
+        Rs_out ([rs.TY_RS_STRICT]): Representation list of output data. Parity must be omitted or set to 0.
+        lmax ([int > 0]): Maximum L used for spherical harmonic in kernel.
```

Comment:
why the notation `([type])`? It confuse me because I read it `a list of objects of type type`

Reply:
Ah, it should just be (type) -- I was using autogenerated doc strings from VS Code and I forgot to delete the extra parenthesis. I will fix and push.